tasks/run-iqe-cji.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
-  name: run-cji-smoke-test
```

Comment:
can we break any user with this change?

Reply:
It will depend on if they are using the git resolver to our pipeline or if they are customizing it. If they are using our pipeline, it should be updated since I am updating the pipeline as well in this PR

Reply:
@gbenhaim no, this shouldn't break users because all of them are referring to the main revision of this repo in their resolvers (except for two apps that Dalia is referring to her fork, but this won't break that either).